snippets/distr-tracing-tempo-tempostack-custom-resource.adoc

```diff
@@ -4,40 +4,49 @@
 // * modules/distr-tracing-tempo-install-tempostack-web-console.adoc
 // * modules/distr-tracing-tempo-install-tempostack-cli.adoc
 
+.Example `TempoStack` CR for AWS S3 and MinIO storage and two tenants
 [source,yaml]
 ----
 apiVersion: tempo.grafana.com/v1alpha1
-kind: TempoStack
+kind: TempoStack # <1>
 metadata:
-  name: sample
+  name: simplest
   namespace: <project_of_tempostack_instance>
 spec:
-  storageSize: <value>Gi # <1>
-  storage:
-    secret: # <2>
-      name: <secret_name> # <3>
-      type: <secret_provider> # <4>
-    tls: # <5>
-      enabled: true
-      caName: <ca_certificate_configmap_name> # <6>
+  storage: # <2>
+    secret: # <3>
+      name: <secret_name> # <4>
+      type: <secret_provider> # <5>
+  storageSize: <value>Gi # <6>
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  tenants:
+    mode: openshift # <7>
+    authentication: # <8>
+      - tenantName: dev # <9>
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa" # <10>
+      - tenantName: prod
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfb"
   template:
+    gateway:
+      enabled: true # <11>
     queryFrontend:
       jaegerQuery:
-        enabled: true
-        ingress:
-          route:
-            termination: edge
-          type: route
-  resources: # <7>
-    total:
-      limits:
-        memory: <value>Gi
-        cpu: <value>m
+        enabled: true # <12>
 ----
-<1> Size of the persistent volume claim for the Tempo WAL. The default is `10Gi`.
-<2> Secret you created in step 2 for the object storage that had been set up as one of the prerequisites.
-<3> Value of the `name` in the `metadata` of the secret.
-<4> Accepted values are `azure` for Azure Blob Storage; `gcs` for Google Cloud Storage; and `s3` for Amazon S3, MinIO, or {odf-full}.
-<5> Optional.
-<6> Optional: Name of a `ConfigMap` object that contains a CA certificate.
-<7> Optional.
+<1> `This CR creates a TempoStack deployment with trace ingestion in the OTLP/gRPC and OTLP/HTTP protocol.` OR `This CR creates a TempoStack deployment, which is configured to receive Jaeger Thrift over the HTTP and OpenTelemetry Protocol (OTLP).` Ruben, a question for you: Is this still Jaeger Thrift? `[Yes/No]` https://pypi.org/project/opentelemetry-exporter-jaeger-thrift/
```

Comment:
@rubenvp8510, :slightly_smiling_face: a question for you on this line ^.

Reply:
yes!

Reply:
@rubenvp8510, :slightly_smiling_face: I'll document it for Jaeger Thrift, but fyi Jaeger Thrift is deprecated:
https://pypi.org/project/opentelemetry-exporter-jaeger-thrift/
![image](https://github.com/user-attachments/assets/ebd2b77c-56fc-431d-a8d5-f3a6034e0301)